docker/devbox/Dockerfile-fluentd

```diff
@@ -0,0 +1,16 @@
+FROM fluent/fluentd:v1.4-onbuild
+
+# Use root account to use apk
+USER root
+
+# below RUN includes plugin as examples elasticsearch is not required
+# you may customize including plugins as you wish
+RUN apk add --no-cache --update --virtual .build-deps \
+        sudo build-base ruby-dev \
```

Comment:
do we need to use `sudo` if we're running as root?

Reply:
We are installing `sudo` tool not running as one :-)
`/bin/sh: sudo: not found`

Reply:
And this is enforced by the base image. I did not diff too deep into the base image recipe

Reply:
PoC in: fa3e51fb62c